StripeCardScan/StripeCardScan/Source/CardScan/UI/VideoFeed.swift

```diff
@@ -242,3 +218,90 @@ class VideoFeed {
         }
     }
 }
+
+// MARK: - Capture Device
+
+private extension VideoFeed {
+    
+    /// Retrieves the most suitable video capture device based on a prioritized list of back camera types.
```

Comment:
I recommend breaking it into smaller one-purpose-only extensions as the current extension is too big to read and navigate with methods that have different purposes.

Reply:
yeah, nice one to help on long-term

Reply:
I am going to do it once his PR gets merged. His PR fix is solid and I see the pain point in having a blur capture.